stripe/src/main/java/com/stripe/android/paymentsheet/ui/BillingAddressView.kt

```diff
@@ -355,6 +356,17 @@ internal class BillingAddressView @JvmOverloads constructor(
         }
     }
 
+    fun populateFromParams(params: PaymentMethodCreateParams) {
```

Comment:
Let's pass an `Address` directly into this method if that's all it cares about. Name can probably be simplified to `populate()`.